app/src/org/commcare/utils/CachingAsyncImageLoader.java

```diff
@@ -110,6 +101,8 @@ public void onLowMemory() {
 
     @Override
     public void onTrimMemory(int level) {
+        // TODO PLM: this is probably never called because Context.registerComponentCallbacks
+        // is never used to register instances of CachingAsyncImageLoader
```

Comment:
@wpride Heads up regarding this code. This is sorta tricky because I'm unsure when the right time is to call  `Context.unregisterComponentCallbacks`. I could also be wrong here.